app/assets/javascripts/validate.js

```diff
@@ -40,6 +40,18 @@ document.addEventListener('turbolinks:load', function() {
               }
             }
             break;
+          case 'phone':
+            if (!value || $.trim(value).length < 1) {
+              notify(this, 'Missing Information');
+            }
+            else if(value){
+              var phoneReg = /^[\+]?[(]?[0-9]{3}[)]?[-\s\.]?[0-9]{3}[-\s\.]?[0-9]{4,6}$/;
```

Comment:
There were a few phone number formats I had issues with. I listed them below with a quick regex refactor that may fix the problem. Additional testing of my regex is needed.

`+1(203)1234567`
`+011234567890`

`^[\+]?[0-9]{0,2}[-\s\.]?[(]?[0-9]{3}[)]?[-\s\.]?[0-9]{3}[-\s\.]?[0-9]{4,6}$`

Reply:
implemented that although made the country code up to 3 digits instead of 2 since many are 3 digits long.